python/lsst/ctrl/execute/libexec/dagIdInfo.py

```diff
@@ -56,3 +57,7 @@
         sys.exit(0)
     file.close()
     sys.exit(0)
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
Probably want all these to be `sys.exit(main())` to match what pip/sconsUtils is going to do with the installable command-lines.